package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mx-react-components",
-  "version": "3.1.33",
+  "version": "3.1.34",
```

Comment:
This was actually supposed to go up with the last release and didn't. I will remove it.